packages/context/src/__tests__/acceptance/interceptor.acceptance.ts

```diff
@@ -208,7 +208,7 @@ describe('Interceptor', () => {
 
     // Listeners added by invocation context are gone now
     // There is one left for ctx.observers
-    expect(ctx.listenerCount('bind')).to.eql(1);
+    expect(ctx.listenerCount('bind')).to.eql(listenerCount + 1);
```

Comment:
Does the number of `BindingEventListener` change alone with the number of the tags of a certain binding and tags only?

Reply:
This test is for number of listeners on a `Context`. Please note that `BindingEventListener` is registered on bindings. When a binding is added to a context, there will be one binding event listener added to the binding by the context.